src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2081,6 +2087,48 @@ private bool RefMustBeObeyed(bool isDelegateCreation, ArgumentSyntax argumentSyn
             return hadError;
         }
 
+        private BoundExpression BindArgumentValue(DiagnosticBag diagnostics, ArgumentSyntax argumentSyntax, bool allowArglist, RefKind refKind)
+        {
+            if (argumentSyntax.Expression != null)
+            {
```

Comment:
I believe, we have decided that tree shapes that are never created by language parser is not something we explicitly guarding against.

---
In reply to: [63748525](https://github.com/dotnet/roslyn/pull/11393#discussion_r63748525) [](ancestors = 63748525)